bids_conversion/convert_somato_data.py

```diff
@@ -0,0 +1,222 @@
+"""Convert the MNE-somato-data to BIDS format.
+
+Requirements
+------------
+The python dependencies to run this script are:
+- numpy
+- scipy
+- mne<0.19
+- mne-bids>0.2
+- nibabel
+
+Notes on freesurfer
+-------------------
+We *cannot* simply copy over the freesurfer files from the non-BIDS somato
+data, because we need to rename the subject to '01' instead of 'somato'. See
+the example workflow below:
+
+$ export SUBJECTS_DIR=/home/username/subjects
+
+$ echo $SUBJECTS_DIR
+/home/username/subjects
+
+$ recon-all -all -i ~/MNE-somato-data-bids/sub-01/anat/sub-01_T1w.nii.gz -s 01
```

Comment:
This will not work if your somato data is not at the same place as you expect. Perhaps it's better to call it from Python with the right paths?

Reply:
This is a comment: I basically expect the user to run **freesurfer** outside of this script. ... Because running a script with freesurfer has proven to keep my computer busy for about 20 hours.

I think most users will let a server do the freesurfer stuff, and then import the data. Then all they need to do is set a path, and run the present script.

Are you suggesting something else?

Reply:
umm ... I was thinking of a script that just works if you run it from start to end. That's kind of the idea behind mne-scripts (see some of the others). Some of the parts take longer times but you don't care if you run it on your office workstation. There are no manual components to it

Reply:
okay, challenge accepted :)